integreat_cms/release_notes/current/unreleased/2574.yml

```
@@ -0,0 +1,2 @@
+en: Add alternative texts for predefined icons
+de: Füge alternative Texte für vordefinierte Ikonen hinzu
```

```suggestion
de: Füge alternative Tags für vordefinierte Icons hinzu
```
Ikonen would be more like the person who is an icon. [Duden](https://www.duden.de/rechtschreibung/Icon) suggests to use just "Icon" :)

---

I had a dilemma about it. Thank you :)